### PR TITLE
PHX-226 Define an internal status of 'second_review_required' when a failing …

### PR DESCRIPTION
**Pull request by chrisndodge**
…review from SoftwareSecure is received. This will help us better identify which reviews need an internal review

### CODE REVIEW

**Comment by chrisndodge** [comment]
@afzaledx can someone from team review?

File: edx_proctoring/api.py [hunk 1]
```diff
@@ -1685,6 +1685,10 @@ def _get_proctored_exam_view(exam, context, exam_id, user_id, course_id):
             ) else 'proctored_exam/submitted.html'
         else:
             student_view_template = 'proctored_exam/waiting_for_app_shutdown.html'
+    elif attempt_status == ProctoredExamStudentAttemptStatus.second_review_required:
+        # the student should still see a 'submitted'
+        # rendering even if the review needs a 2nd review
+        student_view_template = 'proctored_exam/submitted.html'
```
Comment (afzaledx) [inline edx_proctoring/api.py]: @hasnain-naveed, can you update this logic to incorporate PHX-216's changes?

File: edx_proctoring/models.py [hunk 2]
```diff
@@ -196,8 +199,8 @@ def needs_credit_status_update(cls, to_status):
         Returns a boolean if the passed in to_status calls for an update to the credit requirement status.
         """
         return to_status in [
-            cls.verified, cls.rejected, cls.declined, cls.not_reviewed, cls.submitted,
-            cls.error
+            cls.verified, cls.rejected, cls.declined, cls.not_reviewed,
+            cls.submitted, cls.error
```
Comment (afzaledx) [inline edx_proctoring/models.py]: second_review_required not added here, nor in the other lists.

**Comment by afzaledx** [comment]
:+1:

File: edx_proctoring/api.py [hunk 3]
```diff
@@ -1685,6 +1685,13 @@ def _get_proctored_exam_view(exam, context, exam_id, user_id, course_id):
             ) else 'proctored_exam/submitted.html'
         else:
             student_view_template = 'proctored_exam/waiting_for_app_shutdown.html'
+    elif attempt_status == ProctoredExamStudentAttemptStatus.second_review_required:
+        # the student should still see a 'submitted'
+        # rendering even if the review needs a 2nd review
+        student_view_template = None if _was_review_status_acknowledged(
+            attempt['is_status_acknowledged'],
+            exam['due_date']
+        ) else 'proctored_exam/submitted.html'
```
Comment (chrisndodge) [inline edx_proctoring/api.py]: Thanks for adding that

File: edx_proctoring/models.py (hunk 2 above)
Comment (chrisndodge) [inline edx_proctoring/models.py]: yes, that was intentional.

**Pull request merged**